### PR TITLE
tests: run packaging tests against CentOS and Fedora

### PR DESCRIPTION
**Pull request by lebauce**
skip skydive-go-fmt skip skydive-functional-tests-backend-elasticsearch skip skydive-functional-tests-backend-orientdb skip skydive-scale-tests skip skydive-selenium-tests skip skydive-unit-tests skip skydive-compile-tests skip skydive-k8s-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

File: Makefile [hunk 1]
```diff
@@ -407,9 +408,11 @@ docker-image: static
 	cp $$GOPATH/bin/skydive contrib/docker/
 	sudo -E docker build -t ${DOCKER_IMAGE}:${DOCKER_TAG} -f contrib/docker/Dockerfile contrib/docker/
 
-.PHONY: vendor
-vendor: govendor check
-	tar cvzf vendor.tar.gz vendor/
+.PHONY: localdist
+localdist: govendor genlocalfiles
```
Comment (nplanel) [inline Makefile]: why localdist ?  dist do the job from the git repo and avoid tainted situation

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run skydive-packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Comment by lebauce** [comment]
run packaging-tests

**Pull request merged**